tests/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+function(add_oscap_test test_script)
```

Comment:
I suggest 2 functions, add_oscap_shell_test and add_oscap_c_test, the c test also builds and links the test and then adds it.

You can most probably skip the oscap part of the function names unless there is a conflict.

Reply:
@mpreisler I think the C tests are usually wrapped by a bash script, so I would introduce a function that just builds the executable and links it.